docs/tutorials/processing/brain_dataset_tutorial.md

```diff
@@ -0,0 +1,73 @@
+# Using Dask and napari to view and process a high-resolution human brain imaging dataset.
+We used napari to view and process a high-resolution human brain imaging dataset, and in the following tutorial we will break down how it all builds up!
+
+The dataset was obtained from [Dr. Brian Edlow's Lab](https://www.comarecoverylab.org/) for NeuroImaging of Coma and Consciousness, which is dedicated to promoting recovery of consciousness in people with severe brain injuries.
+This dataset is a 100 micron resolution magnetic resonance imaging (MRI) scan of an ex vivo human brain specimen. The brain specimen was donated by a [58-year-old woman](https://www.youtube.com/watch?v=Q-9jzBkoNuI) who had no history of neurological disease and died of non-neurological causes.
```

Comment:
Worth linking to the actual repository too—I think this is the correct link:
https://doi.org/10.5061/dryad.119f80q
and maybe the preprint where this is all described:
https://doi.org/10.1101/649822